data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -367,4 +389,86 @@ private static Schema fileProjection(
 
     return new Schema(columns);
   }
+
+  public static Map<String, PositionDeleteIndex> createPosIndexMap(
+      Iterable<FileScanTask> fileTasks, FileIO io) {
+    return createPosIndexMap(fileTasks, DEFAULT_SET_FILTER_THRESHOLD, io);
+  }
+
+  public static Map<String, PositionDeleteIndex> createPosIndexMap(
+      Iterable<FileScanTask> fileTasks, long filterThreshold, FileIO io) {
+    List<DeleteFile> posDeletes = distinctPosDeletes(fileTasks);
+    if (posDeletes.isEmpty()) {
+      return ImmutableMap.of();
+    }
```

Comment:
Please follow style guidelines. Control flow blocks should be separated from the following statement by a line of whitespace.

Reply:
fixed